src/Diagnostics.DataProviders/Interfaces/IKustoDataProvider.cs

```diff
@@ -11,6 +11,8 @@ public interface IKustoDataProvider : IMetadataProvider
 
         Task<DataTable> ExecuteQuery(string query, string stampName, string requestId = null, string operationName = null);
 
+        Task<DataTable> ExecuteQueryOnAllAntaresClusters(string query, string requestId = null, string operationName = null);
```

Comment:
>ExecuteQueryOnAllAntaresClusters [](start = 24, length = 32)

Is ExecuteQueryOnAppAppServiceClusters better? This is also fine though. Just came to my mind so thought of mentioning.